src/App.js

```diff
@@ -1,32 +1,35 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
-import SearchResults from './context/SearchResults';
-import List from './components/List';
+import MapCenter from './context/MapCenterContext';
+import ListPage from './pages/ListPage';
 import Settings from './components/Settings';
 import Help from './components/Help';
 import Nav from './components/Nav';
-import Map from './components/Map';
+import MapPage from './pages/MapPage';
 
```

Comment:
I like the clarifications of the various file names!

Reply:
That’s a lot of shifting😄. I think we need to keep in mind the app architecture should be easy to navigate and manage,it doesn't exactly feel this way now 😄.